File: bubble sort/bubbleSort.js

numArray = [];
for (let i = 0; i < 100; i++) {
    numArray.push(Math.floor(Math.random() * Math.floor(100)));
}
console.log(numArray)

for (let i = 0; i < numArray.length; i++) {
    for (let j = 0; j < numArray.length; j++) {
        if (j + 1 < numArray.length) {
            if (numArray[j] > numArray[j+1]) {
                temp = numArray[j]
                numArray[j] = numArray[j+1]
                numArray[j+1] = temp
            }
        }
    }
}
console.log(numArray)

File: bubble sort/bubbleSort.py
import random

#Initialising array
numArray = []
for i in range(0, 100):
    numArray.append(random.randint(0,100))
print("Random array")
print(numArray)

#sort
for i in range(0, len(numArray)):
    for j in range(0, len(numArray)):
        if(j+1 < len(numArray)):
            if (numArray[j] > numArray[j+1]):
                temp = numArray[j]
                numArray[j] = numArray[j+1]
                numArray[j+1] = temp

print("Sorted array")
print(numArray)